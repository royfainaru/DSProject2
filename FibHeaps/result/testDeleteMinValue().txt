╮
╰── ╭──────────────────╮
    │ Key: -2147483648 │
    ├──────────────────┤
    │ Rank: 0          │
    │ Marked: false    │
    │ Parent: (null)   │
    │ Next: (null)     │
    │ Prev: (null)     │
    │ Child: (null)    │
    ╰──────────────────╯
