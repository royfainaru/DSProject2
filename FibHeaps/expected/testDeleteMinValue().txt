╮
╰── ╭───────────────────╮
    │ Key: -2147483648  │
    ├───────────────────┤
    │ Rank: 0           │
    │ Marked: false     │
    │ Parent: (null)    │
    │ Next: -2147483648 │
    │ Prev: -2147483648 │
    │ Child: (null)     │
    ╰───────────────────╯
